tests/php/Dev/SapphireTestTest.php

```diff
@@ -271,11 +270,8 @@ public function testEnableErrorHandler(int $errno, ?string $expectedClass): void
     public static function provideEnableErrorHandler(): array
     {
         // Only E_USER_* errors can be triggered, so that's all that's being tested
+        // As of PHP 8.4, E_USER_ERROR can no longer be triggered without a PHP deprecation notice
         return [
-            'error' => [
-                'errno' => E_USER_ERROR,
-                'expectedClass' => ExpectedErrorException::class,
-            ],
```

Comment:
This wasn't mentioned in the release notes but is in the docs: https://www.php.net/manual/en/errorfunc.constants.php#constant.e-user-error
> Usage of this constant with [trigger_error()](https://www.php.net/manual/en/function.trigger-error.php) is deprecated as of PHP 8.4.0. It is recommended to either [throw](https://www.php.net/manual/en/language.exceptions.php) an [Exception](https://www.php.net/manual/en/class.exception.php) or call [exit()](https://www.php.net/manual/en/function.exit.php) instead.